docs/tech-faq.md

```diff
@@ -0,0 +1,42 @@
+---
+comments: true
+---
+
+# FAQ on the technology
+
+On this page, we answer some of the most frequent questions that people have concerning the website etc.
```

Comment:
how about: "..that people have concerning the website and the features of the website." Cutting out the "etc" in this case?